### ISSUE TITLE
Adding cloud Storage 

### ISSUE THREAD

**Issue by oli-mac** [midas-research/audino#138]
hello, can I add an S3 cloud storage like Minio to store and load the date to the annotation tool just like CVAT ?

**Comment by AmanS0007** [comment]
Yes, you can try that on the deployed version at https://app.audino.in , we will merge the changes here also.
